### ISSUE TITLE
While using `webpack --watch`, css file does not rebuild

### ISSUE THREAD

**Issue by akinnee** [LPGhatguy/postcss-sassy-import#17]
After adding postcss-sassy-import, our css bundle no longer updates when we change a css file.

**Comment by LPGhatguy** [comment]
Yeah, this is a problem with the way that imports are currently resolved. 🙁 

Webpack never finds out that we have a dependency on the files that are imported, and so it doesn't know that it needs to watch them for changes.
